go.mod

```
@@ -14,10 +14,11 @@ require (
 	github.com/gorilla/websocket v1.5.0
 	github.com/hyperboloide/lk v0.0.0-20221004131154-cb9733bc66d0
 	github.com/kubeshark/base v0.6.3
-	github.com/rs/zerolog v1.29.0
-	k8s.io/api v0.27.2
-	k8s.io/apimachinery v0.27.2
-	k8s.io/client-go v0.27.2
+	github.com/kubeshark/telemetry v0.0.0-20230824224738-66631a15462e
+	github.com/rs/zerolog v1.30.0
+	k8s.io/api v0.28.1
+	k8s.io/apimachinery v0.28.1
+	k8s.io/client-go v0.28.1
```

```suggestion
	github.com/rs/zerolog v1.29.0
	k8s.io/api v0.27.2
	k8s.io/apimachinery v0.27.2
	k8s.io/client-go v0.27.2
```

Don't change these version with this PR to prevent creating a huge diff in `go.sum`. In order to do that, use the same versions in here: https://github.com/kubeshark/telemetry/blob/master/go.mod#L6-L9

---

my bad, I think it likely got upgraded when running go mod / tidy. Reverted.